profiles/skos-ap-nl.ttl

```diff
@@ -513,7 +551,7 @@ skosapnl:Concept-notation
 skosapnl:Concept-label
   a sh:PropertyShape ;
   rdfs:label "naam"@nl ;
-  sh:datatype xsd:string ;
+  sh:datatype rdf:langString ;
```

Comment:
Oplossing voor issue 7

Reply:
Hey @Bakkej, ik heb je wijziging gecheckt en volgens mij zitten er mooie uit. Je kunt ook mij als reviewer aanmaken, ik weet niet of @fterpstra tijd heeft voor review.